src/cryptoadvance/specter/templates/includes/sidebar/sidebar.jinja

```diff
@@ -106,6 +106,9 @@
             setTimeout(fetchUTXORescanProgress, 1000);
         </script>
         {% endif %}
+        {% if specter.setup_status.stage %}
+            <a href="{{url_for('node_setup_wizard', step=specter.setup_status.stage)}}" class="btn action" style="width: 200px; margin: auto;">See setup progress</a>
```

Comment:
This currently looks quite ugly. I'd suggest:
```suggestion
            {{ sidebar_btn(url_for('node_setup_wizard', step=specter.setup_status.stage), 'See setup progress', class='btn action', icon=false) }}
```

with some modifications in sidebar_btn.jinja
```
{% macro sidebar_btn(url_path, text, id="", icon=true, class="btn") -%}
    <div>
        <a href="{{url_path}}" class="{{class}}" {% if id %}id="{{id}}"{% endif %} style="max-width: 90%;margin: auto;">
            {% if icon %}<svg width="20" height="20" viewBox="0 0 24 24"><path d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"/></svg>{% endif %}{{ text }}
        </a>
    </div>
{%- endmacro %}
```
note the removal of the slash, it was `"{{url_path}}"/` before and the addition of the class param.

Reply:
Done